llvm/lib/Target/AArch64/AArch64.td

```diff
@@ -1351,6 +1382,11 @@ def ProcessorFeatures {
                                  FeatureFP16FML, FeatureSVE, FeatureTRBE,
                                  FeatureSVE2BitPerm, FeatureBF16, FeatureETE,
                                  FeaturePerfMon, FeatureMatMulInt8, FeatureSPE];
+  list<SubtargetFeature> A720 = [HasV9_2aOps, FeatureNEON, FeatureMTE,
```

Comment:
`FeatureNeon`, `FeatureSVE`, `FeatureBF16` and `FeatureMatMulInt8` are already enabled by default on Armv9.2-A. Can they be removed from this list to avoid the redundancy?

Reply:
Also, it looks like the TRM states that FEAT_TME (`FeatureTME`) is not enabled on Cortex-A720.